release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -120,34 +120,35 @@ This release adds improvements related to the following components and concepts.
 [[ocp-311-operator-lifecycle-manager]]
 ==== Operator Lifecycle Manager (Technology Preview)
 
-Operator Lifecycle Manager is currently in
+The Operator Lifecycle Manager (OLM) is currently in
 xref:ocp-311-technology-preview[Technology Preview] and not for production
 workloads.
 
-Run, manage and upgrade many Operators on top of the cluster.
+The OLM aids cluster administrators in installing, upgrading, and granting
+access to Operators running on their cluster.
 
-* There is a catalog of curated Operators.
-* Operator Lifecycle Manager handles rolling updates of all Operators to new versions.
-* There is access control (RBAC) for certain teams to use certain Operators.
+* Includes a catalog of curated Operators, with the ability to load other Operators into the cluster
+* Handles rolling updates of all Operators to new versions
+* Supports role-based access control (RBAC) for certain teams to use certain Operators
 
 See
-xref:../install_config/installing-operator-framework.adoc#whats-in-the-technology-preview[Installing
-the Operator Framework] for more information.
+xref:../install_config/installing-operator-framework.adoc#whats-in-the-technology-preview[Installing the Operator Framework]
+for more information.
 
 [[ocp-311-operator-sdk]]
 ==== Operator SDK
 
-The Operator SDK is a framework designed to make writing operators easier by
-providing:
+The Operator SDK is a development tool to jump-start building an Operator with
+generated code and a CLI to aid in building, testing, and publishing your
+Operator.
 
-* High-level APIs and abstractions to write the operational logic more intuitively.
-* Tools for scaffolding and code generation to quickly bootstrap a new project.
-* Extensions to cover common operator use cases.
+* Provides tools to get started quickly embedding application business logic into an Operator
+* Saves you from doing the work to set up scaffolding to communicate with the Kubernetes API
+* Helps run end-to-end tests of your logic on a local or remote cluster
+* Is used by Couchbase, MongoDB, Redis and more
 
-The Operator SDK provides tools to get started quickly embedding application
-business logic into an Operator. It saves you from doing the work to set up
-scaffolding, helps run end-to-end tests of your logic on a local or remote
-cluster, and it is used by Couchbase, MongoDB, Redis, and more.
+See link:https://docs.okd.io/latest/operators/osdk-getting-started.html[Getting started with the Operator SDK]
+in OKD documentation for more information and walkthroughs.
```

Comment:
@robszumski Linking to ODK doc instead of project README. Can revert if needed.

Reply:
Oh duh, thanks